Tests/fontBuilder/fontBuilder_test.py

```diff
@@ -420,3 +420,49 @@ def test_unicodeVariationSequences(tmpdir):
     fb.setupCharacterMap(cmap, uvs)
     fb.save(outPath)
     _verifyOutput(outPath, tables=["cmap"])
+
+
+def test_setupPanose(is_ttf, keep_glyph_names, make_cff2, post_format):
```

Comment:
you are not using these pytest fixture arguments, just define it as  `def test_setupPanose():`